src/rdfcrate/wrapper.py

```diff
@@ -218,24 +218,32 @@ class AttachedCrate(RoCrate):
     def __post_init__(
         self,
         name: str,
-        description: str,
-        license: str | URIRef,
+        description: str | None,
+        license: str | URIRef | None,
         path: Path | str,
         recursive_init: bool = False,
     ):
         self.root = Path(path)
-        root_dataset = self.register_dir(
-            self.root,
-            recursive=recursive_init,
-            attrs=[
-                (uris.datePublished, Literal(datetime.now().isoformat())),
-                (uris.name, Literal(name)),
-                (uris.description, Literal(description)),
+
+        attrs: list[tuple[URIRef, URIRef | Literal]] = [
```

Comment:
Just realised an even more elegant solution!
```suggestion
        attrs: list[Double] = [
```